packages/context/src/Context/AuthenticationContext.container.js

```diff
@@ -74,10 +86,44 @@ const AuthenticationProviderInt = ({
   useEffect(() => {
     setLogger(otherProps.loggerLevel, otherProps.logger);
     dispatch({ type: 'ON_LOADING' });
-    addOidcEvents(oidcState.userManager.events, dispatch, oidcState.userManager);
+
+    addOidcEvents(
+      oidcState.userManager.events,
+      dispatch,
+      oidcState.userManager,
+      otherProps.onUserLoaded,
+      otherProps.onUserUnloaded,
+      otherProps.onAccessTokenExpiring,
+      otherProps.onAccessTokenExpired,
+      otherProps.onSilentRenewError,
+      otherProps.onUserSignedOut
```

Comment:
Can you put all customEvents in one prop ? And add it in the props list of the component

```suggestion
     customEvents
```